api/firmware/eth.go

```diff
@@ -141,6 +141,17 @@ func (device *Device) handleSignerNonceCommitment(response *messages.ETHResponse
 	return signature, nil
 }
 
+func ETHIdentifyCase(recipientAddress string) messages.ETHAddressCase {
```

Comment:
please add a docstring of the form `// ETHIdentifyCase ...`. All exported API functions should come with docs.